### PR TITLE
Remove Superfluous Docs Tab, & Fix Pages Link

### PR DESCRIPTION
**Pull request by helloimalastair**
This Pull Request addresses the fact that, in my opinion anyway, the Docs Tab appeared to be superfluous. It seems to be a near perfect duplicate of the main page, just with slightly different wording. This PR should address that, by removing the Docs page, and adding the Hero Blocks that were unique to the Docs page to the main page.
An example of what the page may look like when deployed: https://cloudflare-docs-y03x.pages.dev/

### CODE REVIEW

**Comment by helloimalastair** [comment]
Didn't know how you wanted to implement this, so I left it open, but the `/docs` path should probably be redirected back to the main page.

**Comment by helloimalastair** [comment]
Latest commit fixes error with Link in Pages Docs, that leads to 404 on first load.

File: products/pages/src/content/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Get started deploying your first site using our starter guide below, or check ou
 
 ## Popular pages
 
-- [Deploy a React application](/how-to/deploy-a-react-application)
+- [Deploy a React application](/framework-guides/deploy-a-react-application)
```
Comment (kristianfreeman) [inline products/pages/src/content/index.md:19]: could you pull this out into a separate PR?

Comment (helloimalastair) [reply products/pages/src/content/index.md:19]: Done. Accidentally broke this Pull Request, so I had to make a new one. [Pages Link Fix](https://github.com/cloudflare/cloudflare-docs/pull/1471)

**Comment by helloimalastair** [comment]
Accidentally broke this PR, here is the fixed one: [Remove Superfluous Docs Page](https://github.com/cloudflare/cloudflare-docs/pull/1470)
